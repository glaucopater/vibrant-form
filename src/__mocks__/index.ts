import { FormFieldType } from "../components/Vibrant/types";

export const mockFields: FormFieldType[] = [
    {
        label: "labelA",
        value: 100,
    },
    {
        label: "labelB",
        value: 200,
    },
    {
        label: "labelC",
        value: 300,
    },
];