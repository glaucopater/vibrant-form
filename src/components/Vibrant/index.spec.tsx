import VibrantForm from "./";
import { mockFields } from "../../__mocks__/index";

describe("VibrantForm", () => {
  it("should render the empty component", () => {
    expect(VibrantForm).toMatchSnapshot();
  })

  it("should render the component with mock data", () => {
    const wrapper = <VibrantForm fields={mockFields} />;
    expect(wrapper).toMatchSnapshot();
  })
})
