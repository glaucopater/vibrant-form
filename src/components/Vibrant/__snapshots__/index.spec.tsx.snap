// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VibrantForm should render the component with mock data 1`] = `
<VibrantForm
  fields={
    Array [
      Object {
        "label": "labelA",
        "value": 100,
      },
      Object {
        "label": "labelB",
        "value": 200,
      },
      Object {
        "label": "labelC",
        "value": 300,
      },
    ]
  }
/>
`;

exports[`VibrantForm should render the empty component 1`] = `[Function]`;
